readthedocs/proxito/middleware.py

```diff
@@ -173,6 +174,10 @@ def process_request(self, request):  # noqa
         if hasattr(ret, 'status_code'):
             return ret
 
+        if '//' in request.path:
+            # Remove multiple slashes from URL's
+            return redirect(re.sub('//+', '/', request.get_full_path()))
```

Comment:
Just wanted to mention that may be better to use `regex` module if we are applying the regex on user's input. IIRC, there was some security/performance in the `re` module that are not present in `regex`.

Reply:
regex is dangerous when the pattern is arbitrary (ReDOS), here the matching is the one that is arbitrary, so all good.

Reply:
Yea, this regex is super simple, so I don't think it should be an issue.